x-pack/packages/ml/data_grid/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/ml-data-grid
+
+This package contains a custom data grid component that is used by the ML and Transform plugins. On top of `EuiDataGrid` this extended version supports a custom header with mini histograms for each column.
```

Comment:
```suggestion
This package contains a custom data grid component that is used by the ML and Transform plugins. On top of `EuiDataGrid`, this extended version supports a custom header with mini histograms for each column.
```

Reply:
Fixed in [7a447f5](https://github.com/elastic/kibana/pull/155530/commits/7a447f56a22c6bf1ce76549677e6b375786b518f).